formcentral/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM golang:alpine AS build
```

Comment:
I simplified a bit the instructions on the Dockerfile.

I'm curious to know why we use two `FROM` instructions? Was it used to make the docker image size smaller?

Reply:
Yes, that's really the point of a multi-stage build. Why did you want to change it to a single-stage build?

Reply:
That makes sense. I will revert to multi-stage.